### ISSUE TITLE
Instructions missing if Geometry is set to False

### ISSUE THREAD

**Issue by hamzarhaiem** [GIScience/openrouteservice#851]
I’am calling directions with POSt and using this like body

```
{
   "preference":"shortest",
   "units":"km",
   "language":"fr",
   "geometry":"false",
   "instructions":"true",
   "coordinates":[
      [
         6.791700,
         51.999200
      ],
      [
         6.791700,
         53.109200
      ]
   ],
   "options":{
      "avoid_countries":[
         
      ],
      "avoid_borders":"none",
      "avoid_features":[
         
      ],
      "profile_params":{
         "restrictions":{
            "width":0.0,
            "height":0.0,
            "weight":32.0,
            "length":0.0,
            "axleload":3,
            "hazmat":false
         }
      }
   }
}
```

I went to have the response **without geometry**, but even with geometry = false I’am having it on my response. The only way to remove it, is to put **instructions and geometry both to false**, but this will also **remove the segments and the instructions from the response**.

Are there anyway **to have the instructions without the geometry** when i’am routing only two points?

PS: When having more than two points and geometry false and instructions true it worked, i can have the instructions without the geometry.

**Comment by koebi** [comment]
This is caused by [this line](https://github.com/GIScience/openrouteservice/blob/3410e21896c1b9e8d6b85cb1d3db114a0de9e7e2/openrouteservice/src/main/java/org/heigit/ors/routing/RouteResultBuilder.java#L136) in RouteResultBuilder.java, where segments get reset if geometry is set to false.

The fix is straightforward and provided in #863.

I think that there should exist some tests to make sure this doesn't happen again - should this be a Unit-Test or an API-Test?

**Comment by takb** [comment]
I'd guess the quickest would be the API tests for extra info, there should be at least one that goes through that line
